pkgs/by-name/mi/min-ed-launcher/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildDotnetModule,
+  fetchFromGitHub,
+  git,
+}:
+buildDotnetModule rec {
+  pname = "min-ed-launcher";
+  version = "0.11.3";
+
+  src = fetchFromGitHub {
+    owner = "rfvgyhn";
+    repo = "min-ed-launcher";
+    tag = "v${version}";
+    hash = "sha256-s+E7DvM2XpfCz6oIeI/Na5NSxN8gHnzWzABXOrPlwLk=";
```

Comment:
This hash needs changing to `sha256-HJIvbuTsCG51PPVieJbXGyAviqgM9/WPz0+0VhIWz9k=`

Reply:
I updated it but I see that it was the same one (I did an empty one so nix would generate it for me).

I was finally able to reproduce the new has by rebasing on master (should've thought of this sooner).